### PR TITLE
Create test for POST photo api. Add code for post_photo_api in serial…

### PR DESCRIPTION
**Pull request by fieldstyler**
Create test for POST photo api. 
Add code for post_photo_api in serializers, urls, and views

Co-authored-by: Norma Lopez <norma_l304@yahoo.com>

# Pull Request

## What's this PR do?

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Testing
- [ ] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?

Create functioning POST request to create a photo. Endpoint: api/v1/weddings/create_photo

## What are the relevant tickets?
* Addresses
* Closes

### CODE REVIEW

File: app/wedding/serializers.py [hunk 1]
```diff
@@ -13,3 +14,9 @@ class GuestSerializer(serializers.ModelSerializer):
     class Meta:
         model = Guest
         fields = ('id', 'name', 'phone_number', 'wedding')
+
+
+class PhotoSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Photo
+        fields = ('id', 'number', 'description', 'guest')
```
Comment (IamNorma) [inline app/wedding/serializers.py:22]: Looks good here

File: app/wedding/tests/test_photo_api.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                        date='2345675432'
+                        )
+        wedding.save()
+
+        guest = Guest.objects.create(
+                        name='Uncle Bob',
+                        phone_number='2343454567',
+                        wedding=wedding
+        )
+        guest.save()
```
Comment (IamNorma) [inline app/wedding/tests/test_photo_api.py:34]: I believe .create saves the object, so guest.save() might not be necessary here

File: app/wedding/tests/test_photo_api.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        photo_data = {
+            'number': '1',
+            'description': 'Bride with Groom Family',
+            'guest': guest.id
+        }
+
+        res = self.client.post(CREATE_PHOTO_URL, photo_data)
+        # code.interact(local=locals())
```
Comment (IamNorma) [inline app/wedding/tests/test_photo_api.py:43]: This line can be deleted

**Review by IamNorma — CHANGES_REQUESTED** [review]

File: app/wedding/tests/test_photo_api.py (hunk 2 above)
Comment (fieldstyler) [reply app/wedding/tests/test_photo_api.py:34]: Deleted it and test still passes. Good catch.

**Review by IamNorma — APPROVED** [review]

**Pull request merged**